File: CMS/src/app/shared/CompanyInfoDonViDongGoiSanPham.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { CompanyInfoDonViDongGoiSanPham } from 'src/app/shared/CompanyInfoDonViDongGoiSanPham.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class CompanyInfoDonViDongGoiSanPhamService extends BaseService{

    List: CompanyInfoDonViDongGoiSanPham[] | undefined;
    ListFilter: CompanyInfoDonViDongGoiSanPham[] | undefined;
    FormData!: CompanyInfoDonViDongGoiSanPham;

    DisplayColumns001: string[] = ['STT', 'Name', 'Note', 'Save'];

    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "CompanyInfoDonViDongGoiSanPham";
    }
}

File: Service/Implement/CompanyInfoDonViDongGoiSanPhamService.cs
namespace Service.Implement
{
    public class CompanyInfoDonViDongGoiSanPhamService : BaseService<CompanyInfoDonViDongGoiSanPham, ICompanyInfoDonViDongGoiSanPhamRepository>
    , ICompanyInfoDonViDongGoiSanPhamService
    {
    private readonly ICompanyInfoDonViDongGoiSanPhamRepository _CompanyInfoDonViDongGoiSanPhamRepository;
    public CompanyInfoDonViDongGoiSanPhamService(ICompanyInfoDonViDongGoiSanPhamRepository CompanyInfoDonViDongGoiSanPhamRepository) : base(CompanyInfoDonViDongGoiSanPhamRepository)
    {
    _CompanyInfoDonViDongGoiSanPhamRepository = CompanyInfoDonViDongGoiSanPhamRepository;
    }
    }
    }

